### PR TITLE
John conroy/tutorial changes

### LINKED ISSUE #1413: Resolve Chuck's comments on datasets search tutorial
The comments are from #1411.

1.
> Little thing: There's still an animation down from the upper left corner of the window... I think it's distracting, but I could be off-base?

Neither Tiffany nor I could reproduce a tutorial tooltip transition animation.

2.
> Bigger thing: I worry that there are steps that read like calls-to-action, but if the user actually does that, the tutorial just exits. Like step 2: "Click the View All button to display the entire list of filters in the selected category." If I actually do that, pouf! -- no more tutorial.

The toggle tile view step no longer instructs or requires the user to click the button and instead relies on a next button. We should revisit the language and whether clicking the highlighted element should end the tutorial.

3.
> Does the library support using the forward and back keys to move through slides?

It doesn't, but since we're using a custom tooltip we could easily add this. Is it the correct decision for accessibility?

4.
> @tsliaw : Is there an idiom for "The End" slides? I read the last slide as another call to action, and when I clicked, the tutorial going away was a surprise.

@tsliaw in response:

> Given that the "Next" button becomes "Last" and (6/6) is displayed, I think what is currently displayed should be good enough for now. We can revisit this later if this becomes an issue.

5.
> Where does the app bar get its z-index from? Should it be added to list?

App bar gets its z-index from MUI. We could overwrite that value if we'd prefer it to come from our code.

### PR DESCRIPTION
**Pull request by john-conroy**
Fix #1413. Closing #1414 since we need to keep the tutorial controlled for the keydown listeners.

@mccalluc confirmed the animation is not present so no change needed there.

### CODE REVIEW

**Review by mccalluc — COMMENTED** [review]
Looks good. A couple tweaks, if they seem reasonable to you.

**Comment by tsliaw** [comment]
When I press back from Step 5 to Step 4, the tutorial exits for me with no way for me to continue the tutorial unless I refresh the page.

File: context/app/static/js/components/tutorials/SearchDatasetTutorial/SearchDatasetTutorial.jsx [hunk 1]
```diff
@@ -28,34 +28,34 @@ const defaultSteps = [
...
+      'To further narrow the relevant datasets, type search terms or phrases in “quotes" into the Search bar. Datasets containing any of the search terms will be returned.',
     disableBeacon: true,
     title: 'Search Datasets by Free Text',
   },
   {
     target: 'div.sk-layout__results.sk-results-list > table > thead > tr > th:nth-child(3)',
-    content: `Press the arrow button by the relevant column to sort search results. A bolded arrow indicates the current sorting selection.
-      ${' '}Click again to reverse the order.`,
+    content: `Clicking the arrow button by the relevant column will sort search results. A bolded arrow indicates the current sorting selection.
+    ${' '}Clicking again will reverse the order.`,
```
[the remark this answers is not in the document]
Comment (john-conroy) [reply context/app/static/js/components/tutorials/SearchDatasetTutorial/SearchDatasetTutorial.jsx:38]: Is this trouble worth avoiding the long string?

[the remark this answers is not in the document]
Comment (john-conroy) [reply context/app/static/js/components/tutorials/SearchDatasetTutorial/SearchDatasetTutorial.jsx]: Works, thanks.

**Comment by john-conroy** [comment]
@tsliaw fixed, thanks.

**Pull request merged**